FreeRTOS/Test/CMock/makefile.in

```diff
@@ -41,20 +41,24 @@ CPPFLAGS    :=
 
 # Compiler flags
 CFLAGS              :=  $(INCLUDE_DIR)  -O0 -ggdb -pthread  --std=c99 -Werror -Wall
-CFLAGS              +=  -fsanitize=address,undefined -fsanitize-recover=address
 CFLAGS              +=  -fstack-protector-all
 CFLAGS              +=  -Wformat -Werror=format-security -Werror=array-bounds
 CFLAGS              +=  -D_FORTIFY_SOURCE=2
+ifdef ENABLE_SANITIZER
```

Comment:
Will this get enabled if `ENABLE_SANITIZER=0` is passed?

Reply:
Changed to ifeq ( $(ENABLE_SANITIZER),1)